website/content/docs/enterprise/ent-to-ce-downgrades.mdx

```diff
@@ -0,0 +1,219 @@
+---
+
+# Downgrade from Consul Enterprise to the community edition
+
+This document describes how to downgrade from Consul Enterprise to Consul community edition (CE).This is only applicable from version 1.18 onwards for both Consul Enterprise and Consul community edition (CE).
+
+## Download the CE binary version
+
+First, download the binary for the CE version(>=1.18).
+
+<Tabs>
+<Tab heading="Binary">
+
+All current and past versions of the CE and Enterprise releases are
+available on the [HashiCorp releases page](https://releases.hashicorp.com/consul)
+
+Example:
+```shell-session
+$ export VERSION=1.18.0
+$ curl https://releases.hashicorp.com/consul/${VERSION}/consul_${VERSION} _linux_amd64.zip
+```
+
+</Tab>
+<Tab heading="Kubernetes">
+
+To downgrade consul version on k8s we will have to modify the image version in your helm chart similar to the upgrade process mentioned in this doc.
+  [Upgrade Consul version](https://developer.hashicorp.com/consul/docs/k8s/upgrade#upgrade-consul-version)
+
+</Tab>
+
+</Tabs>
+
+## Prepare for the downgrade to CE
+
+**1.** Take a snapshot:
+
+```shell-session
+$ consul snapshot save backup.snap
+```
+
+You can inspect the snapshot to ensure if was successful with:
+
+```shell-session
+$ consul snapshot inspect backup.snap
+```
+
+Example output:
+
+```
+ID           2-1182-1542056499724
+Size         4115
+Index        1182
+Term         2
+Version      1
+```
+
+This will ensure you have a safe fallback option in case something goes wrong. Store
+this snapshot somewhere safe. More documentation on snapshot usage is available here:
+
+- [Consul snapshot](/consul/commands/snapshot)
+- [Backup Consul Data and State tutorial](/consul/tutorials/production-deploy/backup-and-restore)
+
+**2.** Temporarily modify your Consul configuration so that its [log_level](/consul/docs/agent/config/cli-flags#_log_level)
+       is set to `debug`. After doing this, issue the following command on your servers to
+       reload the configuration:
+
+```shell-session
+$ consul reload
+```
+
+This change will give you more information to work with in the event something goes wrong.
+
+**3.** Modify the following config entries if applicable: 
+
+[Service resolver](https://developer.hashicorp.com/consul/docs/connect/config-entries/service-resolver): 
+
+  1. Remove services configured as failovers in non default namespace or  belonging to a sameness group.
+
+  2. Remove services configured as redirects belonging to non default namespaces or partiton.
+
+[Service splitter](https://developer.hashicorp.com/consul/docs/connect/config-entries/service-splitter):
+
+  1. Remove services configured as splits belonging to non default namespaces or partiton
+
+[Service router](https://developer.hashicorp.com/consul/docs/connect/config-entries/service-router):
+
+  1. Remove services configured as destination belonging to non default namespaces or partiton
```

Comment:
Is it possible to indent this section for formatting

Reply:
done